.github/workflows/acceptance-tests-runner.yml

```diff
@@ -648,6 +648,8 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.aws_access_key_id }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.aws_secret_access_key }}
           AWS_REGION: ${{ vars.AWS_REGION }}
+          AWS_REGION_UPPERCASE: ${{ vars.AWS_REGION_UPPERCASE }}
+          AWS_REGION_LOWERCASE: ${{ vars.AWS_REGION_LOWERCASE }}
```

Comment:
question: was there a reason for adding AWS_REGION_UPPERCASE and AWS_REGION_LOWERCASE, vs using AWS_REGION and using existing function `conversion.MongoDBRegionToAWSRegion`?